db_stress_tool/db_stress_gflags.cc

```diff
@@ -534,6 +534,11 @@ DEFINE_int64(
     preserve_internal_time_seconds, 0,
     "Preserve internal time information which is attached to each SST.");
 
+DEFINE_uint32(use_timed_put_one_in, 0,
+              "If greater than zero, TimedPut is used per every N write ops on "
+              "on average. (N is an approximation, the actual average would be "
```

Comment:
Isn't this "actual average would be higher" stuff obsolete? (Here and below)

Reply:
Thank you for catching this.